lib/test/test_broadcasting.py

```diff
@@ -0,0 +1,51 @@
+from apytypes import APyFixedArray, APyFixed, APyFloatArray, APyFloat
+
+import math
+import pytest
+from typing import List, Tuple
+
+# TODO: Remove me
+import numpy as np
```

Comment:
I think you can use this type of test, just check the other NumPy tests and do import-or-skip.